src/app/views/common/copy/KeyboardCopyEvent.ts

```diff
@@ -0,0 +1,40 @@
+import { eventTypes, telemetry } from '../../../../telemetry';
+import { KEYBOARD_COPY_TABS } from '../../../../telemetry/component-names';
+
+interface IComponentList {
+  [key: string]: string;
+}
+export const KeyboardCopyEvent = () => {
+  const componentList: IComponentList = KEYBOARD_COPY_TABS;
+  document.addEventListener('keydown', (event: KeyboardEvent) => {
+    if (event && (event.ctrlKey || event.metaKey) && event.key === 'c') {
```

Comment:
Assigning 'c' here limits it to cases where a user has their Caps Lock off... 
We could use the key code/key value of 'c', which is 67, to include all cases...

Reply:
The event.keyCode property was deprecated. We can use event.key.toLowerCase() to cover both cases where Caps Lock is on or off
![image](https://user-images.githubusercontent.com/45680252/220310458-34d8d736-a3e1-420c-b741-aa1bf6fe2495.png)

Reply:
Aah okay perfect!